### PR TITLE
Delete DCIP folder.

### PR DESCRIPTION
**Pull request by rowanc1**
I have deleted the DCIP folder.

Can @sgkang and @fourndo confirm that all functions have been moved to EM/Statics?

### CODE REVIEW

**Comment by fourndo** [comment]
Confirmed.

**Comment by lheagy** [comment]
@sgkang, @fourndo : would you guys mind taking care of the merge conflicts here? If you run into anything major, we can sit together and deal with it

File: SimPEG/Examples/DC_Forward_PseudoSection.py [hunk 1]
```diff
@@ -74,7 +74,13 @@ def run(loc=None, sig=None, radi=None, param=None, surveyType='dipole-dipole', u
 
     # We will handle the geometry of the survey for you and create all the combination of tx-rx along line
     # [Tx, Rx] = DC.gen_DCIPsurvey(locs, mesh, surveyType, param[0], param[1], param[2])
-    survey, Tx, Rx = DC.gen_DCIPsurvey(locs, mesh, surveyType, param[0], param[1], param[2])
+    srcList = StaticUtils.gen_DCIPsurvey(locs, mesh, surveyType, param[0], param[1], param[2])
+    print(srcList)
+
+    raise Exception('The output of the function changed. We need to create the survey and get Rx to continue.')
```
Comment (rowanc1) [inline SimPEG/Examples/DC_Forward_PseudoSection.py]: @fourndo @sgkang Can you fix up this example? The code has changed, maybe you know what to?

Comment (lheagy) [inline SimPEG/Examples/DC_Forward_PseudoSection.py]: @fourndo : is this example tricky to fix? We could remove it from this pr and add it back later if so, or just fix it on this pr. Let us know what is best!

Comment (fourndo) [inline SimPEG/Examples/DC_Forward_PseudoSection.py]: @lheagy : Yea it s the best way right now. I will keep it local and bring it back in once I find the time to fix it.
@seogi : I think you and I already talked about this. You can remove from PR

**Pull request merged**